usl_pipeline/cloud_functions/main.py

```diff
@@ -16,12 +16,12 @@
 import xarray
 import rasterio
 
+import usl_lib.shared.wps_data as wps_data
```

Comment:
should move this between
from usl_lib.readers import elevation_readers
&
from usl_lib.storage import cloud_storage

to keep the import alphabetized. I'm surprised there isn't a lint for this.